docs/omero_figure_move.rst

```diff
@@ -0,0 +1,99 @@
+
+#.  Inside OMERO.figure, click ``File > Open`` and open the Figure you wish to move to another group.
+
+#.  Click ``File > Move Figure to Group...``. In the following dialog, select the group you wish to move the Figure to and click OK.
+
+#.  Observe a dialog reporting a success of the move. Verify that when you now again click ``File > Move Figure to Group...`` the Figure is now reported to be in your intended group.
+
+.. note::
+      You might want to move the images contained in the Figure to the other group as well. This step is optional, if you want to use your figures by yourself only, but you should consider it in case the users you want to share your figure with do not have permissions to see your images in the original group (for example they are members of the target group only).  
+
+.. _Retrieval:
+
+*Retrieval of images contained in figures using a script*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#.  Go to OMERO.figure and open the Figure. Copy the last part of the url from the address bar of your browser, which contains the Figure ID, e.g.:
+
+    https://your-omero-server.org/web/figure/file/79828
+
+    has the Figure ID ``79828``.
+
+#.  Repeat the step above for all the Figures containing the images you want to move. Retain the Figure IDs (for example saving them as a text document) for later use.
+
+#.  Go back to the tab with OMERO.web, create a new Dataset and select it.
+
+#.  Click on the ``Scripts`` icon |image1| above the central pane of OMERO.web. Select ``Figure scripts > Figure Images to Dataset``. Start the script and paste into the dialog which appears your Figure IDs, separated with commas and no spaces, e.g. ``79828,79830,71228``. Click ``Run``.
+
+#.  When the script finishes, refresh the page and find the images contained in your Figure linked to the Dataset. Note that the images are only doubly-linked to two Datasets now, and no independent copy of the images was done in this step.
+
+#.  The above means that when you execute the Move into another group, the images in your Dataset will no longer be available in the original group.
+
+#.  Move the images into the group you have moved the Figure to. This will enable any user permitted to see your images in the target group to view your Figure.
+
+*Duplication of images, move and re-create Figure(s) in the new group using a script*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case you want to have both Figures and the images contained in them in both groups, the original and the target group, you can use following workflow. Note that this workflow involves usage of Command Line.
+
+#.  Retrieve the images contained in the Figures using the workflow above. For this, you can find out the Figure IDs either manually as described in the chapter :ref:`Retrieval<Retrieval>` or by running hql queries on the command line, such as::
```

Comment:
Instead of "Retrieve" I think "Collect the Images contained in the Figures into a single Dataset using the workflow above".

Reply:
fixed in https://github.com/ome/omero-guide-figure/pull/13/commits/4eb8f01b2bd0f64dcb0ae49158bc6592b117988c